src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final Color DEFAULT_COLOR = Color.WHITE;
```

Comment:
why public? would make sense to move it to your parameter class, same as DEFAULT_RADIUS